utilities/DumpUnitaryTest/UnitTest1.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Xunit;
+using Quantum.DumpUnitary;
+
+namespace Test
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void Test1()
```

Comment:
I'd use a name with some indication of what test it is - this way we could (hypothetically) add more tests to cover other patterns without getting confused.

```suggestion
        public void BlockDiagonalTest()
```